### PR TITLE
Evo: add output to a json file option when running on a video

### PR DESCRIPTION
**Pull request by tdurand**
_Based on the detector.c code that does the same for a single image_

This add an option when running on a video to output all results in a JSON file

`./darknet detector demo cfg/coco.data cfg/yolov3.cfg yolov3.weights file.mp4 -dont_show -json_file_output result.json`

Not sure if you want to merge this, but at least I have it shared with people that would want such feature

I updated the README also

### CODE REVIEW

**Comment by haviduck** [comment]
This works as described and have helped tremendously in my work, the stored json has all detected objects. but somehow neither my console or my preview window show objects. Is this just me messing up the build?

**Comment by tdurand** [comment]
Yes, you should uncomment the line:

```
//draw_detections_cv_v3(show_img, local_dets, local_nboxes, demo_thresh, demo_names, demo_alphabet, demo_classes, demo_ext_output);
```

And run the command without the flag `-dontshow`

**Comment by cenit** [comment]
looks like i don't have permission to update PR to latest master. 
https://github.com/opendatacam/darknet/pull/22 should do the trick

File: src/coco.c [hunk 1]
```diff
@@ -397,7 +397,8 @@ void run_coco(int argc, char **argv)
     float hier_thresh = find_float_arg(argc, argv, "-hier", .5);
     int cam_index = find_int_arg(argc, argv, "-c", 0);
     int frame_skip = find_int_arg(argc, argv, "-s", 0);
-    int ext_output = find_arg(argc, argv, "-ext_output");
+	int ext_output = find_arg(argc, argv, "-ext_output");
```
Comment (cenit) [inline src/coco.c:400]:
```suggestion
    int ext_output = find_arg(argc, argv, "-ext_output");
```

**Pull request merged**